packages/core/src/input/Input.css

```diff
@@ -185,4 +185,17 @@
   display: inline-flex;
   padding-left: var(--salt-size-adornmentGap);
   column-gap: var(--salt-size-adornmentGap);
-}
\ No newline at end of file
+}
+
+/* TODO: Firefox fix */
+.saltInput:has(.saltInput-startAdornmentContainer):has(.saltAdornmentButton) {
+  --saltInput-paddingLeft: calc(0.25 * var(--salt-size-unit));
```

Comment:
makes sense to use `--salt-spacing-25`?
same for all `--salt-size-unit` calculations here